docs/infra/style-guide.md

```diff
@@ -54,6 +55,54 @@ Here are some exceptions (and additions) to Hashicorp's Terraform style guide.
 
 - For policy enforcement and compliance checks, [Tfsec](https://github.com/aquasecurity/tfsec) is used instead of [Terraform's policy enforcement framework](https://developer.hashicorp.com/terraform/cloud-docs/policy-enforcement)
 
+## GitHub Actions style
+
+- Use short job names with at most 3 words to improve readability within the GitHub UI. Long job names get cut off with an ellipsis (…).
+- Use imperative phrases for step names. Examples: "Build and publish", "Run migrations", "Set up Terraform".
+- Use a single whitespace character for job names when calling reusable workflows. This is because in pull requests and in the workflow visualization, GitHub shows the job name as `<job name in calling workflow> / <job name in reusable workflow>` and the calling workflow job name is usually redundant. For example:
+
+    ```yaml
+    build-and-publish:
+      name: " "
```

Comment:
I want to see a before / after on this one before I endorse it, because it's awkward to look at. Plus I would assume that removing this field entirely would have the same effect?

Reply:
removing this field would default to the job id e.g. "build-and-publish" (they might do some automatic formatting like stripping dashes i don't remember)

added before/after screenshots to the description

Reply:
(I can't really show "before" for checks in pull requests because those were so painful before that I had already fixed all of those for a while, so i don't know of "Before" examples for those